lib/src/controls/surfaces/expander.dart

```diff
@@ -198,6 +198,52 @@ class ExpanderState extends State<Expander>
   Widget build(BuildContext context) {
     assert(debugCheckHasFluentTheme(context));
     theme = FluentTheme.of(context);
+    if (widget.content == null) {
```

Comment:
Would this be some boilerplate code? Isn't it better to check if content is null on `SizeTransition`?

Reply:
You might be right, but I haven't used `SizeTransition` before. Would the use of `SizeTransition` still hide the handle and prevent any actions on the object? (Hover color etc)